lua/nvim-biscuits/init.lua

```diff
@@ -140,34 +140,15 @@ nvim_biscuits.decorate_nodes = function(bufnr, lang)
     end
 end
 
-nvim_biscuits.setup = function(user_config)
```

Comment:
I moved this function below `nvim_biscuits.BufferAttach` definition so that we can use it inside the setup.